release-team/role-handbooks/branch-manager/README.md

```diff
@@ -181,3 +182,16 @@ The current documentation in the [contributor guide for cherry picks](https://gi
 The cherry pick script is also fairly self documenting in terms of how to invoke the command.
 
 There has been quite a bit of recent discussion (see: [1](https://github.com/kubernetes/community/pull/2408), [2](https://github.com/kubernetes/community/pull/1980)) around improving both the cherry pick process process and its documentation.  Watch for and contribute to improvements.
+
+# Staging Repositories
+
+The [publishing-bot](https://github.com/kubernetes/publishing-bot) is responsible for publishing the code in
+[staging](https://git.k8s.io/kubernetes/staging/src/k8s.io) to their own repositories.
+
```

Comment:
can we list the repos the bot publishes to or is it too large to list. If not is there a link to the list?

Reply:
Is there an ETA of when the publishing happens? i.e., within hours of the new tag appearing or within days? it will be good to clarify the expectation here.

Reply:
> can we list the repos the bot publishes to or is it too large to list.

There are 19 published repos. :)

> If not is there a link to the list?

The link at the word `staging` points to the list of  staging directories that are published.

> Is there an ETA of when the publishing happens? i.e., within hours of the new tag appearing or within days? it will be good to clarify the expectation here.

Good point, I have updated it. :+1:  Just to give some more background about tihs:

Right now, the bot runs every 4 hours, so tagging can have a delay of at most 4 hours (if all is good with the bot).

The client-go major release (eg `v1.13.0`) is manual and is released a day after the main release because the main release occurs around 5pm PST and both @sttts (Germany) and I (India) are not in the US. We run smoke tests the day after (in our timezone) and release it then.